src/Eppie.App/Eppie.App.ViewModels/BaseAccountSettingsPageViewModel.cs

```diff
@@ -140,6 +140,10 @@ protected virtual async Task ApplySettingsAndGoBackAsync()
             catch (NeedAdditionalAuthInfo)
             {
             }
+            catch (AuthorizationException)
```

Comment:
make it more specific: `catch (AuthorizationException ex) when (ex.Message == "Proton: invalid mailbox password")`

Reply:
Fixed.